packages/db-schema/src/utils/drop-all.ts

```diff
@@ -0,0 +1,23 @@
+import { sql } from "drizzle-orm";
+import { isPgSchema } from "drizzle-orm/pg-core";
+
+import schema from "../index.js";
+import { Database } from "../types.js";
+
+export async function dropAll(db: Database) {
```

Comment:
consider renaming this to `dropAllSchemas`?  Otherwise its not really clear anymore what "all" is being dropped